src/ApiService/IntegrationTests/.gitignore

```diff
@@ -1,3 +0,0 @@
-__azurite_db_*.json
```

Comment:
why deleted?

Reply:
I moved it to the top level gitignore with matching on any subfolders. I think the location of those output files can change.